packages/local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -12,8 +12,8 @@ const launchChrome = require('../util/launchChrome');
 
 const {exec} = require('child_process');
 
-function launchChromeDevTools(host, args = '') {
-  var debuggerURL = 'http://' + host + '/debugger-ui' + args;
+function launchChromeDevTools(host, port, args = '') {
+  var debuggerURL = `http://${host}:${port}/debugger-ui${args}`;
```

Comment:
(nit) you could probably turn this into a `const` while this part is being modified

Reply:
Oh, this "suggested change" is nice!

Reply:
@Esemesek please resolve this one and we will ship!

Reply:
@grabbou it's fixed in https://github.com/react-native-community/react-native-cli/pull/28/commits/6a727c3567ac210c60e45a5ac71c05eb9a58c201

Reply:
Interesting why it doesn't say "dimissed" then

Reply:
Github UI changed a bit and I think it doesn't auto-fold the outdated conversations, unless you mark them so, it just adds the `Outdated` label at the top of the diff.